.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Launch Program",
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
+            "program": "${workspaceFolder}/js/index.js"
+        }
+    ]
+}
```

Comment:
Not a big deal, but I _think_ you can delete this file, since it's the one for running js files with node. But if it's working with this file, then just ignore me 😅